_drgn.pyi

```diff
@@ -733,6 +733,41 @@ class Program:
         :param lang: :attr:`Type.language`
         """
         ...
+    def threads(self) -> ThreadIterator:
+        """
+        Return an iterable over all of the threads resident within the process.
+        """
+        ...
+    def thread(self, tid: int) -> Thread:
+        """
+        Retrieve the thread with the given `tid`, raising a `LookupError` in the
```

Comment:
Sphinx has a `raises` directive for documenting exceptions. Look for `:raises` in this file for examples.

Please also add a `param` directive for `tid`. Even when the description is somewhat obvious, it at least adds the type annotation to the documentation.